README.md

```diff
@@ -463,4 +463,47 @@ def ==(other)
   other.is_a?(Quantity) && ...
   # ...or, sometimes, duck type
   other.respond_to?(:amount) && other.respond_to?(:unit) && ...
+end
+```
+
+## Pattern Matching
+
+Value objects are a prime candidate for use with the pattern matching syntax introduced in Ruby 2.7.
+
+The easiest way to integrate with pattern matching is to return a hash of attributes from `#decontruct_keys`, which you have probably already implemented in `#to_h`. This allows the object to be pattern matched using the hash syntax:
+
+```ruby
+class Quantity
+  ...
+
+  def decontruct_keys(_keys)
+    to_h
+  end
+end
+
+case Quantity.new(10, 'm')
+in { unit: 'm', amount: }
```

Comment:
a) a personal one: the rest of README uses "no spaces inside hash braces" style (my personal); maybe not the most common, but probably should be consistent throughout?.. Though,
b) maybe just remove the braces at all?

Reply:
I'm still getting used to the new syntax, so I didn't know the brackets were optional. I've removed them now.